Source/Diagnostics/FlushFormats/FlushFormat.H

```diff
@@ -27,7 +27,10 @@ public:
         bool isLastBTDFlush = false,
         const amrex::Vector<int>& totalParticlesFlushedAlready = amrex::Vector<int>() ) const = 0;
 
-     virtual ~FlushFormat() {}
+    virtual ~FlushFormat() {}
+
+    /** Write general info of the run into the plotfile */
+    void WriteJobInfo(const std::string& dir) const;
```

Comment:
@dpgrote I wonder if this should be written by full diagnostics at all - or instead rather like the `warpx_used_inputs` file just in general, when the simulation starts?

Reply:
Good question. I like this kind of thing to be written out in the same place as the diagnostics so that it is associated with the output of the simulation. If this is written at the startup, the code would have to figure out where the diagnostics will be written.